run_unittests.py

```diff
@@ -4827,6 +4827,10 @@ def test_pkgconfig_gen(self):
         self.assertEqual(foo_dep.get_pkgconfig_variable('foo', {}), 'bar')
         self.assertPathEqual(foo_dep.get_pkgconfig_variable('datadir', {}), '/usr/data')
 
+        libhello_nolib = PkgConfigDependency('libhello_nolib', env, kwargs)
+        self.assertTrue(libhello_nolib.found())
+        self.assertEquallibhello_nolib.get_link_args(), [])
```

Comment:
typo here.

Also test that libhello_nolib.get_compile_args() is empty to be complete.